src/eth/methods/methods.ts

```diff
@@ -53,7 +53,7 @@ export class EthMethod implements Method {
         types.push(tuple);
         const abiElement = 'function ' + this.name + '(' + tuple + ')';
         const iface = new ethers.utils.Interface([abiElement]);
-        const methodId = iface.functions[this.name].sighash;
+        const methodId = iface.getSighash(iface.functions[this.name])
```

Comment:
Seems unit tests are failing. Some of them are already failing due to #40. the pipeline may not be configured correctly unfortunately.

Reply:
hey just checkout to this commit https://github.com/BitGo/smart-contracts/pull/40/commits/6dae62b075548d97c706161e86ce0bd00e69a0a0 and testcases are passing until this point
I believe this pr https://github.com/BitGo/smart-contracts/pull/44/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519L4133 introduced major ether library upgrade hence tests are failing now

Reply:
adding `"ethers": "4.0.48"` in packages.json and using `iface.functions[this.name].sighash;` worked for me

Reply:
Done